packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,33 @@
+# 47.6.0 - November 14, 2019
+
+## Fixed
+
+#### docs
+
+- Fix broken links for LWC [local development](https://forcedotcom.github.io/salesforcedx-vscode/articles/lwc/localdev) ([PR #1769](https://github.com/forcedotcom/salesforcedx-vscode/pull/1769))
+
+- Fix typo in `SFDX: Refresh SObject Definitions` command ([PR #1772](https://github.com/forcedotcom/salesforcedx-vscode/pull/1772))—Contribution by [@eouellette](https://github.com/eouellette)
+
+## Added
+
+#### salesforcedx-vscode-apex
+
+- Remove beta disclaimer from semantic error description of Apex Language Server ([PR #1785](https://github.com/forcedotcom/salesforcedx-vscode/pull/1785))
```

Comment:
@rcoringrato-sfdc , (not sure but) would you consider adding this and the next entry (capitalize template options...) in the Fixed section? Thanks.